scripts/bootstrap_cluster.sh

```diff
@@ -119,10 +119,62 @@ install_argocd_helm_chart() {
     --set controller.extraArgs[0]='--application-namespaces="*"'
 }
 
+install_argo_charts() {
+    local cluster_type=$1
+    local argo_charts=('argocd' 'argo-rollouts')
+
+    for argo_chart in "${argo_charts[@]}"; do
+        response=$(curl "https://catalogue.truefoundry.com/$cluster_type/templates/$argo_chart.yaml")
+        echo "$response" > /tmp/application.yaml
+        
+        kubectl apply -f /tmp/application.yaml -n argocd
+        rm -f /tmp/application.yaml
+    done
+}
+
+install_istio_dependencies() {
+    local cluster_type=$1
+    local istio_dependencies=('istio-base' 'istio-discovery' 'tfy-istio-ingress');
+
+    for istio_dependency in "${istio_dependencies[@]}"; do
+        response=$(curl "https://catalogue.truefoundry.com/$cluster_type/templates/istio/$istio_dependency.yaml")
```

Comment:
Add `--silent` here in the `curl` command. You can maybe add an echo command to print what we are installing.